h2o-automl/src/main/java/water/automl/api/schemas3/EventLogV99.java

```diff
@@ -0,0 +1,30 @@
+package water.automl.api.schemas3;
```

Comment:
renamed from UserFeedbackV99

Reply:
@seb-h2o when the file name gets changed it is a tough task to check what was really changed :)

Reply:
you can check the old ones, but that's why I write additional changes on top file: this one is pure renaming

Reply:
@seb-h2o I got your intents.. just wanted to express my compassion :)